db/schema.rb

```diff
@@ -30,13 +30,15 @@
   end
 
   create_table "posts", force: :cascade do |t|
-    t.bigint "user_id", null: false
+    t.integer "user_id"
     t.string "title", limit: 255
     t.text "text"
     t.integer "comments_counter", default: 0, null: false
     t.integer "likes_counter", default: 0, null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "author_id"
+    t.string "author"
     t.index ["user_id"], name: "index_posts_on_user_id"
   end
```

Comment:
- [ ] You have `user_id`, `author_id` and `author` columns on the posts table, but that should not be so. I would recommend you run new migrations and remove `author_id` and `author`, then rename `user_id` to `author_id`.

  The reason I ask you not to maintain the initial `author_id` and instead rename `user_id` is so that we can have an index on the `author_id` foreign key